Pinta.Core/Classes/Palette.cs

```diff
@@ -159,15 +161,44 @@ public void LoadDefault ()
 
 		public void Load (string fileName)
 		{
-			var loader = PintaCore.System.PaletteFormats.GetLoaderByFilename (fileName);
-			colors = loader.Load (fileName);
-			colors.TrimExcess ();
-			OnPaletteChanged ();
+			try
+			{
+				var loader = PintaCore.System.PaletteFormats.GetLoaderByFilename (fileName);
+				colors = loader.Load (fileName);
+				colors.TrimExcess ();
+				OnPaletteChanged ();
+			}
+			catch (NullReferenceException e)
```

Comment:
Maybe `GetLoaderByFilename()` should be changed to just return `null` if the format doesn't exist, similar to `ImageFormats.GetImporterByFile()`? It's a bit weird that it throws a NullReferenceException there

Reply:
Done. Changed the code to the style similar to ```WorkspaceManager.OpenFile()``` method. I hope it is correct this time.